### ISSUE TITLE
GeoIP2 Database Error

### ISSUE THREAD

**Issue by lolmanz0** [LostTeam/LostCore#2]
"The file "GeoIP2-Country.mmdb" does not exist or is not readable." (EXCEPTION) in "/67-LostCore_v0-346e5/src/MaxMind/Db/Reader" at line 45

I know it is because there is not database. But I do not know how to create this database. Please, can you help me? This is not like MySQL. So, how to make this database? Thank you!

**Comment by thelucyclub** [comment]
create a file with that name then write `return(null);` in it.

**Comment by lolmanz0** [comment]
Okay, I will test it. Thanks.

**Comment by thelucyclub** [comment]
in newer pmmp builds, that doesn't work. I'll try and update the repo if i have time.

**Comment by lolmanz0** [comment]
Okay, great!

**Comment by thelucyclub** [comment]
What version are you on?

**Comment by thelucyclub** [comment]
of the plugin

**Comment by lolmanz0** [comment]
1.0.0

**Comment by lolmanz0** [comment]
From this site.

**Comment by thelucyclub** [comment]
i cant extract it...

**Comment by thelucyclub** [comment]
lemme try something

**Comment by thelucyclub** [comment]
nope. i cant extract the phar so i cant update it. :/

**Comment by lolmanz0** [comment]
Do you want to help with it?

**Comment by lolmanz0** [comment]
Yeah, there is a problem with decompile. Can you update the pre-release version please? I think it is the same. Try it.

**Comment by thelucyclub** [comment]
no i cant. i cant use it.

**Comment by lolmanz0** [comment]
But sou can use pre-release 0.0.13.

**Comment by lolmanz0** [comment]
You*

**Comment by jasonwynn10** [comment]
This plugin was never finished. It never reached an operable state.

**Comment by thelucyclub** [comment]
@jasonwynn10 true, but the website says it should work flawlessly. You are false advertising

**Comment by thelucyclub** [comment]
But since this isn't exactly a "real issue" atm, it may as be closed.

**Comment by jasonwynn10** [comment]
IT SAYS NO WHERE THAT IT SHOULD WORK AT ALL

**Comment by thelucyclub** [comment]
LEL RIGHT DERE
![557454e](https://cloud.githubusercontent.com/assets/10909225/24824111/46a5bd5a-1bd4-11e7-83de-9435b7de32a6.PNG)

**Comment by jasonwynn10** [comment]
That is API specific. Not for developmental versions of PocketMine
